### PR TITLE
IBUS telemetry new features

### PR DESCRIPTION
**Pull request by hali9**
Support for:
10ch_Mavlink_MOD_i6_Programmer_V1
fix_updater_03_16_21_33
fix_updater_03_16_21_33_1
independent sensor type and value
Pitot sensor
presure sensor
climb sensor
gps status sensor
4 byte long LAT, LON, ALT with formating
armed sensor
mode sensor
speed in km\h or m\s

### CODE REVIEW

File: src/main/telemetry/ibus_shared.c [hunk 1]
```diff
@@ -121,68 +130,134 @@ static ibusAddress_t getAddress(uint8_t ibusPacket[static IBUS_MIN_LEN]) {
 }
 
 // MANUAL, ACRO, ANGLE, HRZN, ALTHOLD, POSHOLD, RTH, WP, LAUNCH, FAILSAFE
-static uint8_t flightModeToIBusTelemetryMode[FLM_COUNT] = { 0, 1, 3, 2, 5, 6, 7, 4, 8, 9 };
-
+static uint8_t flightModeToIBusTelemetryMode1[FLM_COUNT] = { 0, 1, 3, 2, 5, 6, 7, 4, 8, 9 };
+static uint8_t flightModeToIBusTelemetryMode2[FLM_COUNT] = { 5, 1, 0, 7, 2, 8, 6, 3, 4, 9 };
```
Comment (digitalentity) [inline src/main/telemetry/ibus_shared.c:132]: Why do we need second set of values?

Comment (hali9) [reply src/main/telemetry/ibus_shared.c:132]: flightModeToIBusTelemetryMode2 is for Mode sensor.
it is realy diffrent and can display text.

**Comment by digitalentity** [comment]
@hali9 if you have the time - can you please rebase?

**Comment by hali9** [comment]
@digitalentity done.

**Comment by digitalentity** [comment]
@hali9 merged!

**Pull request merged**